extensions/opc/src/fetchopc.cpp

```diff
@@ -32,153 +32,136 @@
 
 namespace org::apache::nifi::minifi::processors {
 
-  void FetchOPCProcessor::initialize() {
-    setSupportedProperties(Properties);
-    setSupportedRelationships(Relationships);
-  }
+void FetchOPCProcessor::initialize() {
+  setSupportedProperties(Properties);
+  setSupportedRelationships(Relationships);
+}
 
-  void FetchOPCProcessor::onSchedule(core::ProcessContext& context, core::ProcessSessionFactory& factory) {
-    logger_->log_trace("FetchOPCProcessor::onSchedule");
+void FetchOPCProcessor::onSchedule(core::ProcessContext& context, core::ProcessSessionFactory& factory) {
+  logger_->log_trace("FetchOPCProcessor::onSchedule");
 
-    translatedNodeIDs_.clear();  // Path might has changed during restart
+  translated_node_ids_.clear();  // Path might has changed during restart
 
-    BaseOPCProcessor::onSchedule(context, factory);
+  BaseOPCProcessor::onSchedule(context, factory);
 
-    nodeID_ = context.getProperty(NodeID).value_or("");
-    std::string value = context.getProperty(NodeIDType).value_or("");
+  node_id_ = context.getProperty(NodeID).value_or("");
```

Comment:
Since NodeID is required we can safely parse it like this
```suggestion
  node_id_ = utils::parseProperty(context, NodeID);
```

Reply:
Updated in https://github.com/apache/nifi-minifi-cpp/pull/1940/commits/e417d8adfcb004daeaacefe0b1fe9b384505cf79